UPGRADE-3.0.md

```diff
@@ -758,6 +758,38 @@ UPGRADE FROM 2.x to 3.0
  * The `supportsAttribute()` and `supportsClass()` methods of classes `AuthenticatedVoter`, `ExpressionVoter`
    and `RoleVoter` have been removed.
 
+ * The `AbstractVoter::supports()` method signature have changed: second argument now is object instead of class name.
```

Comment:
Minor grammar thing: 
`The `AbstractVoter::supports()` method signature has changed: second argument is now an object instead of class name.`